### ISSUE TITLE
Scalability

### ISSUE THREAD

**Issue by brettwooldridge** [kaitoy/sneo#3]
One more question.  Was sneo designed for scalability testing?  For example, if I wanted to simulate 5-10,000 SNMP devices?

Thanks.

**Comment by kaitoy** [comment]
No. I've never tested SNeO from the performance perspective nor run it for such huge networks.
I've ever used it to simulate 20-30 devices on a PC with 4GB of memory.
